agent/cmd/determined-agent/run.go

```diff
@@ -152,10 +155,5 @@ func newRunCmd() *cobra.Command {
 	cmd.Flags().StringVar(&opts.Fluent.ContainerName, "fluent-container-name", "determined-fluent",
 		"Name for the Fluent Bit container")
 
-	cmd.Flags().IntVar(&opts.AgentReconnectAttempts, "agent-reconnect-attempts",
```

Comment:
can you please bring these back? I used to use these as command-line flags sometimes for debugging

Reply:
yeah, was removed by mistake - sorry, haven't self reviewed this far yet